Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -255,9 +273,10 @@ class MainTabBarViewController: UITabBarController {
                 txHash: txHash,
                 safe: safe
```

Comment:
we need to call completion (onRetry) also when the 'error' screen is swiped down or closed without tapping on "retry" button. In that case we should just remove the safe.